src/apps/content-editor/src/app/views/Analytics/components/AnalyticsPropertySelector.tsx

```diff
@@ -11,6 +11,8 @@ type Props = {
 };
 
 const removeProtocolAndSubdomain = (url: string): string => {
+  if (!url) return "";
```

Comment:
How is this function used? Will it expect an empty string? Should this rather throw an error? i.e. what should the return contract on this function be?

Reply:
Also what value was being feed into this function causing the error? Is that value an acceptable value to be provided?

Reply:
@shrunyan Actually I wasn't able to replicate this even when I logged in to the client's instance that this issue happened with on Sentry. The function is just used to format a url that's from an api call to the analytics api. I assumed that the function received an undefined value due to it missing from the response that's why I decided to just return an empty string.

The function renders this text
![image](https://github.com/user-attachments/assets/17272e29-76f0-42ac-bf58-14ec3016ae6d)